<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hospital_Manage</class>
 <widget class="QMainWindow" name="Hospital_Manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hospital Manage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hospitals</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_hospitals">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>831</width>
      <height>411</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>411</height>
     </size>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Edit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Delete</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_import">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab_search_add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>941</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_search">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>811</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Search Doctor</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_search">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_search">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>90</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search Hospital</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_add_new">
     <attribute name="title">
      <string>Add new</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phone:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_name">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>31</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_phone">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_address">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_description">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="action_config"/>
   </widget>
   <widget class="QMenu" name="menuPrograms">
    <property name="title">
     <string>Programs</string>
    </property>
    <addaction name="action_doctor"/>
    <addaction name="action_patient"/>
    <addaction name="action_schedule"/>
   </widget>
   <addaction name="menuSetting"/>
   <addaction name="menuPrograms"/>
  </widget>
  <action name="action_config">
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionDoctor">
   <property name="text">
    <string>Doctor</string>
   </property>
  </action>
  <action name="actionPatient">
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="actionSchedule">
   <property name="text">
    <string>Schedule</string>
   </property>
  </action>
  <action name="actionDoctor_2">
   <property name="text">
    <string>Doctor</string>
   </property>
  </action>
  <action name="actionPatient_2">
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="actionSchedule_2">
   <property name="text">
    <string>Schedule</string>
   </property>
  </action>
  <action name="action_doctor">
   <property name="text">
    <string>Doctor</string>
   </property>
  </action>
  <action name="action_patient">
   <property name="text">
    <string>Patient</string>
   </property>
  </action>
  <action name="action_schedule">
   <property name="text">
    <string>Schedule</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
